--- uv.gyp	2015-09-22 22:12:07.000000000 +0200
+++ uv.gyp~	2016-04-04 18:22:10.126238144 +0200
@@ -171,7 +171,7 @@
               # This will cause gyp to set soname
               # Must correspond with UV_VERSION_MAJOR
               # in include/uv-version.h
-              'product_extension': 'so.1',
+              'product_extension': 'so.<(soname_version)',
             }],
           ],
         }],
