--- uv.gyp~	2016-08-11 09:29:32.754362526 +0100
+++ uv.gyp	2016-08-11 09:30:15.262533581 +0100
@@ -171,7 +171,7 @@
               'link_settings': {
                 # Must correspond with UV_VERSION_MAJOR and UV_VERSION_MINOR
                 # in src/version.c
-                'libraries': [ '-Wl,-soname,libuv.so.0.10' ],
+                'libraries': [ '-Wl,-soname,libuv.so.<(soname_version)' ],
               },
             }],
           ],
